packages/ag-charts-enterprise/src/components/dialog/dialog.ts

```diff
@@ -72,28 +72,42 @@ export abstract class Dialog<Options extends DialogOptions = DialogOptions> exte
     /**********
      * Inputs *
      **********/
-    protected createButtonGroup({ label, options, value, onChange }: ButtonGroupOptions) {
+    protected createRadioGroup({ label, options, value, onChange }: RadioGroupOptions) {
         const group = this.createInputGroup(label);
+        group.role = 'radiogroup';
+        group.tabIndex = -1;
+        group.ariaLabel = label;
 
         const activeClass = 'ag-charts-dialog__button--active';
+        const buttons: HTMLButtonElement[] = [];
 
         for (const button of options) {
             const buttonEl = createElement('button', `ag-charts-dialog__button`);
             const iconEl = createElement('span', this.ctx.domManager.getIconClassNames(button.icon));
+            buttonEl.role = 'radio';
+            buttonEl.ariaLabel = this.ctx.localeManager.t(button.altText ?? '');
+            buttonEl.ariaChecked = 'false';
+            iconEl.ariaHidden = 'true';
             if (button.value === value) {
                 buttonEl.classList.add(activeClass);
```

Comment:
I think this needs `buttonEl.ariaChecked = 'true';` in this if block, rather than the `buttons[0]` check below?

Reply:
Good suggestion. Done https://github.com/ag-grid/ag-charts/commit/8b172f2527ff39edc32a900605b0681e3b34de15